<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videoGUI</class>
 <widget class="QMainWindow" name="videoGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>videoGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="fileNameLabel">
              <property name="text">
               <string>文件名：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fileNameEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="fileOpenBtn">
              <property name="text">
               <string>打开</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="imageLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="splitProcessBtn">
            <property name="text">
             <string>切分</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelBtn">
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closeBtn">
            <property name="text">
             <string>关闭</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="messageLabel">
            <property name="text">
             <string>消息</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="messageTextBrowser"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="fileOpenAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>动静态算法</string>
     </property>
     <addaction name="frameDifAlgAction"/>
     <addaction name="multiThrhldFramedifAction"/>
     <addaction name="vibeAlgAction"/>
    </widget>
    <addaction name="setOutputAction"/>
    <addaction name="displayAction"/>
    <addaction name="menu_4"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="versionAction"/>
    <addaction name="usedToolAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="fileOpenAction">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="setOutputAction">
   <property name="text">
    <string>输出</string>
   </property>
  </action>
  <action name="displayAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示</string>
   </property>
  </action>
  <action name="versionAction">
   <property name="text">
    <string>版本</string>
   </property>
  </action>
  <action name="frameDifAlgAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>帧间差分算法</string>
   </property>
  </action>
  <action name="multiThrhldFramedifAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>多阀值帧间差分算法</string>
   </property>
  </action>
  <action name="vibeAlgAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>VIBE算法</string>
   </property>
  </action>
  <action name="usedToolAction">
   <property name="text">
    <string>开发工具</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
